Database/MongoDB/Query.hs

```diff
@@ -1518,7 +1841,8 @@ type Command = Document
 
 runCommand :: (MonadIO m) => Command -> Action m Document
 -- ^ Run command against the database and return its result
-runCommand c = fromMaybe err <$> findOne (query c "$cmd") where
+runCommand c = do
+  fromMaybe err <$> findOne (query c "$cmd") where
```

Comment:
nit: It looks like this code didn't really change. Let's keep only the necessary changes.

Reply:
yep. will change that.